src/routers/InitializeSessionRouter.ts

```diff
@@ -0,0 +1,35 @@
+import { Request } from 'express';
+import { OAuth2Client, LoginTicket } from 'google-auth-library';
+
+import ApplicationRouter from '../appdev/ApplicationRouter';
+
+import UserSessionRepo from '../repos/UserSessionRepo';
+import { SerializedUserSession } from '../common/types';
```

Comment:
I think it looks cleaner if we put all of the named imports above the default imports